app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -31,9 +31,14 @@
     <% end %>
     <% if history.any? %>
       <div class="change-notes" data-module="toggle">
-        <p>
+        <noscript>
+          <%# The following p appears only if javascript is disabled %> 
+          <p style="margin-bottom: 5px;">Full page history:</p>
+        </noscript>
+        <p id="full-history-link" style="display: none;">
           <a href="#full-history" data-controls="full-history" data-expanded="false">+ full page history</a>
         </p>
+        <script>$("#full-history-link").show();</script>
```

Comment:
@fofr @dsingleton, I tried to render the behaviour we want, but if you have better ideas on where I should include the js which shows the full-history-link and how to modify the "Full page history:" title style, just let me know. I tried to see it on a reader, but this whole section seems to be hidden...